### ISSUE TITLE
SigMF Needs a Website

### ISSUE THREAD

**Issue by jacobagilbert** [gnuradio/SigMF#169]

**Comment by Teque5** [comment]
Can we draw the website with an oscillioscope?

**Comment by bhilburn** [comment]
So, in all seriousness, I think the oscilloscope video should be on the website.

Also, I think the website should be simple. Really, just a landing page that describes what SigMF is, why you would want it, and how to start using it.

A good go-to example of this, I think, is the spacemacs website. I'm not actually a spacemacs user, but have always remembered how simple and useful their landing page is:

https://www.spacemacs.org/

Since it could be static, Github could host it for us. We just need to actually build the content. Any volunteers? 😄

**Comment by 777arc** [comment]
Could also include a list of SigMF extensions

**Comment by Nepomuceno** [comment]
Could that be a github pages ( the easier way to host a simple website here ) and it would be reliable and the source for the websice would live side by side with the specs ( can be a different repo but under the same org ) I would help build it just need someone to create the repo should be fairly straigfoward to create a basic one.

**Comment by Nepomuceno** [comment]
It could also host the reference specs that way we could referecne them correctly in the schema files. ( it would be very helpfult o have a proper domain too )

**Comment by 777arc** [comment]
Yeah github pages is great, I use it to host pysdr.org, I can definitely set up the page.io repo if someone throws together a draft website.
